### DISCUSSION TITLE
CycleGAN 9epoch result v2

### DISCUSSION THREAD

**Discussion by Dongbin-Lee-git** [bcaitech1/p4-fr-soccer#16] [Experiment]
##  Data convert to GrayScale
- 앞의 글과 다른 점 : 동일 trained model을 다른 inference condition으로 진행
---
###  OpenCV Filtering result (위) vs CycleGAN result (아래)

![image](https://user-images.githubusercontent.com/57521132/119634209-79ca6a80-be4d-11eb-845c-2710316d7d08.png)![image](https://user-images.githubusercontent.com/57521132/119633822-12141f80-be4d-11eb-8e67-cd06e8e24089.png)
---
![image](https://user-images.githubusercontent.com/57521132/119633946-3243de80-be4d-11eb-87fe-d0fb7a070db2.png)
![image](https://user-images.githubusercontent.com/57521132/119634163-69b28b00-be4d-11eb-8b29-c024a9f49cdb.png)
---
![image](https://user-images.githubusercontent.com/57521132/119634520-cc0b8b80-be4d-11eb-9257-e4f282b73ce8.png)
![image](https://user-images.githubusercontent.com/57521132/119634617-dcbc0180-be4d-11eb-802e-112b8d64998c.png)
---
![image](https://user-images.githubusercontent.com/57521132/119634755-037a3800-be4e-11eb-992d-bbc952ccb7c2.png)
![image](https://user-images.githubusercontent.com/57521132/119634812-0f65fa00-be4e-11eb-9651-ac50ccbe86e2.png)
---
![image](https://user-images.githubusercontent.com/57521132/119634932-2ad10500-be4e-11eb-8137-501a27160827.png)
![image](https://user-images.githubusercontent.com/57521132/119634997-37edf400-be4e-11eb-948c-865a90a35f35.png)

**Comment by karim-moon** [comment]
배경 지우고 글자만 남기는거 진짜신기....
